frontend/public/components/utils/resource-log.tsx

```diff
@@ -185,14 +208,42 @@ export const LogControls: React.FC<LogControlsProps> = ({
       </Tooltip>
     );
   };
-
+  const label = t('public~Debug container');
   return (
     <div className="co-toolbar">
       <div className="co-toolbar__group co-toolbar__group--left">
         <div className="co-toolbar__item">{showStatus()}</div>
         {dropdown && <div className="co-toolbar__item">{dropdown}</div>}
-
         <div className="co-toolbar__item">{logTypeSelect(!hasPreviousLog)}</div>
+        {!showDebugAction(resource, containerName) && podPhase(resource) === 'CrashLoopBackOff' && (
```

Comment:
I wouldn't use `podPhase` here. The problem is if there are multiple containers and one is crash looping while another isn't, you're not guaranteed to get `CrashLoopBackOff` back from `podPhase`.

Reply:
Ok, it was working for me to identify overall if the pod had any crash loop errors.  I will create a new function that solely looks for the crash loop in a pod.